config-laptop.plist/kaby-lake.md

```diff
@@ -92,7 +92,11 @@ This section is allowing spaces to be pass-through to macOS that are generally i
 ### Quirks
 
 ::: tip Info
-Settings relating to boot.efi patching and firmware fixes, for us, we leave it as default
+Settings relating to boot.efi patching and firmware fixes, for most users, leave it as default.
+
+* **ProtectMemoryReigons**: YES
```

Comment:
Regions is misspelled, should be `ProtectMemoryRegions`. Think the same mistake is made down below